tealer/teal/teal.py

```diff
@@ -49,6 +51,10 @@ def instructions(self) -> List[Instruction]:
     def bbs(self) -> List[BasicBlock]:
         return self._bbs
 
+    @property
+    def subroutines(self) -> List[List["BasicBlock"]]:
```

Comment:
We should add a comment on what subroutines return, and mention that this is empty if the version <4

Reply:
sounds good, will do that